Revise list of applications shown to showcase good applications

lionirdeadman opened the pull request:
The criterias were determined in https://github.com/flatpak/flatpak.github.io/issues/489
Fixes https://github.com/flatpak/flatpak.github.io/issues/489

Commit:
Revise list of applications shown to showcase good applications

The criterias were determined in https://github.com/flatpak/flatpak.github.io/issues/489
Fixes https://github.com/flatpak/flatpak.github.io/issues/489

mwleeds commented on data/apps.yml:
```diff
@@ -1,119 +1,71 @@
-  summary:  "3D modelling and animation"
-  color:    "#D89C7F"
-  "flatpakref": "https://flathub.org/repo/appstream/com.slack.Slack.flatpakref"
+- name:     "Flatseal"
```
I think the target audience for flatpak.org is mostly people who are new to Flatpak, and the target audience for Flatseal is advanced users, so maybe not a good fit?

lionirdeadman replied:
This currently won't show because the frontend code limits to 7 applications. I think showcasing Flatseal is still a good idea even if a bit iffy because it risks breaking things.

Worth also noting that none of these applications are clickable so it's likely that someone would only recognize the icon rather than actively click on it to install it.

mwleeds replied:
Maybe flathub.org is a better place to showcase Flatseal than flatpak.org

mwleeds commented:
Added a comment to #489 with some feedback on this

Commit:
Remove Flatseal

This is done because Flatseal is deemed to be for people with intermediate
knowledge of Flatpak therefore it would be unwise to showcase it to the
general public.

AsciiWolf commented:
As mentioned [here](https://github.com/flatpak/flatpak.github.io/issues/489#issuecomment-945033174), GIMP Flatpak seems to be maintained by upstream and I don't think that removing it from the list is a good idea since it is a pretty popular application and the only one Linux native and Free Software app that can compete with Adobe Photoshop.

nanonyme commented on data/apps.yml:
```diff
@@ -1,119 +1,63 @@
+    "128":  "https://flathub.org/repo/appstream/x86_64/icons/128x128/org.gnome.Builder.png"
+    "64":   "https://flathub.org/repo/appstream/x86_64/icons/64x64/org.gnome.Builder.png"
 
-- name:     "GIMP"
```
Yeah, must not be merged until explained more thoroughly which criteria is violated.

AsciiWolf commented:
Any update?

TheEvilSkeleton commented:
I can take over this MR

lionirdeadman commented:
> I can take over this MR

Go ahead :+1:

AsciiWolf commented:
Ping?

TheEvilSkeleton commented:
Err, I completely forgot 😅
